video/getting-started/create-video-client/create-video-client.js

```diff
@@ -7,4 +7,11 @@
 //     const Video = Twilio.Video;
 //
 
-const client = new Video.Client(accessToken);
+// 1.0.0-beta4 and earlier
+//const client = new Video.Client(accessToken);  
+
+// 1.0.0-beta5 onwards the Client class has been removed. 
+// Instead of constructing a Client using an Access Token and then calling connect 
+// on it, you can simply call connect and pass it an Access Token directly.
+const room = await Twilio.Video.connect('your-token');
```

Comment:
Since we introduce `Video` as `const Video = require('twilio-video')` or `const Video = Twilio.Video` above, I think we should just say

```js
const room = await Video.connect('your-token')
```